XMLExporter throws StringIndexOutOfBoundsException if headerText of dataTable is empty ("")

## 1) Environment
- PrimeFaces version: 6.0.8
- Does it work on the newest released PrimeFaces version? Version? Should not work on newer versions
- Does it work on the newest sources in GitHub? (Build by source -> https://github.com/primefaces/primefaces/wiki/Building-From-Source) Should not work with latest source code
- Application server + version:
- Affected browsers:

## 2) Expected behavior

Column with empty header is exported correctly

## 3) Actual behavior
Throws StringIndexOutOfBoundsException

## 4) Steps to reproduce
You need a datatable with one column (<p:column headerText="">) and it is exported with
<p:dataExporter type="xml" target="table" fileName="#{cc.attrs.fileName}" />

Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: 0
	at java.lang.AbstractStringBuilder.charAt(AbstractStringBuilder.java:237) [rt.jar:1.8.0_112]
	at java.lang.StringBuilder.charAt(StringBuilder.java:76) [rt.jar:1.8.0_112]
	at org.primefaces.util.XMLUtils.escapeTag(XMLUtils.java:64) [primefaces-6.0.8.jar:6.0.8]
	at org.primefaces.component.export.XMLExporter.getColumnTag(XMLExporter.java:121) [primefaces-6.0.8.jar:6.0.8]
	at org.primefaces.component.export.XMLExporter.exportCells(XMLExporter.java:94) [primefaces-6.0.8.jar:6.0.8]
	at org.primefaces.component.export.Exporter.exportRow(Exporter.java:265) [primefaces-6.0.8.jar:6.0.8]
	at org.primefaces.component.export.Exporter.exportAll(Exporter.java:250) [primefaces-6.0.8.jar:6.0.8]
	at org.primefaces.component.export.XMLExporter.export(XMLExporter.java:52) [primefaces-6.0.8.jar:6.0.8]
	at org.primefaces.component.export.DataExporter.processAction(DataExporter.java:120) [primefaces-6.0.8.jar:6.0.8]

## 5) Workaround
Use <p:column headerText=" "> instead of <p:column headerText="">


## 6) Poss. Fix
In XMLUtils.java:

Row 63ff
 // Make sure the first character is an allowed one
        if (sb. !XMLChar.isNameStart( sb.charAt(0)) ) {
            sb.insert(0, '_' ); 
        }
    
=>

 // Make sure the first character is an allowed one
        if (sb.length()>0 && sb. !XMLChar.isNameStart( sb.charAt(0)) ) {
            sb.insert(0, '_' ); 
        }
    
 

..

--- a/src/main/java/org/primefaces/util/XMLUtils.java
+++ b/src/main/java/org/primefaces/util/XMLUtils.java
@@ -31,7 +31,7 @@ public class XMLUtils {
      * @return valid XML element name
      */
     public static String escapeTag(String intag) {
-        if ( XMLChar.isValidName(intag) ) {
+        if ( XMLChar.isValidName(intag) || intag == null || intag.length() == 0) {
             return intag;
         }
         

